<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNATDialogBase</class>
 <widget class="QDialog" name="GNATDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GNaT</string>
  </property>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOutputFolder">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>100</y>
     <width>466</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A folder or directory which where tool outputs will be written. The outputs include:&lt;/p&gt;&lt;p&gt;- stream network line shapefile, with network IDs added as a new attribute field&lt;/p&gt;&lt;p&gt;- stream network nodes point shapefile, representing nodes within the network&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseOutput">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>97</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtInputNetwork">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>45</y>
     <width>466</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Shapefile representing a stream network which serves as the primary input for the Build Network tool.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseInput">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>42</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInput">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>25</y>
     <width>186</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input stream network shapefile:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOutput">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>80</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Processing output folder:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProcess">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>145</y>
     <width>146</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Process stream network:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResultsTxt">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Processing results:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExportToTxt">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>450</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDisplayResults">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>530</y>
     <width>116</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInstructions">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>465</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Instructions&lt;/span&gt;:&lt;/p&gt;&lt;p&gt;Clicking &lt;span style=&quot; font-weight:600;&quot;&gt;Display Results&lt;/span&gt; will load the processed stream network into the current QGIS map legend and display area. The user can then manually perform edits to the shapefile to fix incorrectly disconnected sub-networks. Additional help is available in the &lt;a href=&quot;#&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;wiki&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtResults">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>222</y>
     <width>511</width>
     <height>217</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
